### PR TITLE
beautify font size adjustment

### PR DESCRIPTION
**Pull request by crustypluto19**
- Made animation smoother
- Changed font color

https://user-images.githubusercontent.com/24537483/214120785-38c0c19d-7112-49f9-852d-e82f34058a5a.mp4

### CODE REVIEW

**Comment by huber-florian** [comment]
In my option the "Aa" is too close to the score diagram. What do you think? Maybe you should keep one of the paddings that you deleted.

**Comment by botrufus** [comment]
In my opinion the animation still looks quite buggy. I think having a an overlayed small "bubble" like the confirmation dialog that opens would look way nicer. So basically like the menu that opens when pressing the Back button but smaller and custom made of course.

**Comment by pal03377** [comment]
I would also love to see a bubble as described by @botrufus!

**Comment by crustypluto19** [comment]
> In my option the "Aa" is too close to the score diagram. What do you think? Maybe you should keep one of the paddings that you deleted.

Yes I accidentally removed it, will add it back.

**Comment by crustypluto19** [comment]
> I would also love to see a bubble as described by @botrufus!

This wasn't in the problem statement (at least how I understood it), but will gladly do it 👍

**Comment by crustypluto19** [comment]
Changes from the latest commit (now with popover):


https://user-images.githubusercontent.com/24537483/214156412-24a458a7-0537-4b0b-8a7e-86e4be0d4725.mp4

File: Themis/Views/AppearanceSettings/EditorFontSizeStepper.swift [hunk 1]
```diff
@@ -24,31 +24,34 @@ struct EditorFontSizeStepperView: View {
 
     var body: some View {
         HStack {
-            Image(systemName: "textformat.size")
-                .onTapGesture {
-                    withAnimation(.easeInOut(duration: 0.6)) {
-                        showStepper.toggle()
-                    }
+            Button(action: decrementFontSize) {
+                Label("", systemImage: "minus")
```
Comment (AlexExcellence) [inline Themis/Views/AppearanceSettings/EditorFontSizeStepper.swift]: Here, I would rather use Image(systemName: "minus")

File: Themis/Views/AppearanceSettings/EditorFontSizeStepper.swift [hunk 2]
```diff
@@ -24,31 +24,34 @@ struct EditorFontSizeStepperView: View {
...
+                    ensureLargeEnoughFontSize()
                 }
-                .foregroundStyle(showStepper ? .yellow : .gray)
+                .foregroundColor(colorScheme == .light ? .black : .white)
+                .keyboardType(.numberPad)
+                .fixedSize()
+                .frame(width: 30)
+            
+            Button(action: incrementFontSize) {
+                Label("", systemImage: "plus")
```
Comment (AlexExcellence) [inline Themis/Views/AppearanceSettings/EditorFontSizeStepper.swift]: same here

**Review by AlexExcellence — APPROVED** [review]
Looks great!

File: Themis/Views/Assessment/AssessmentView.swift [hunk 3]
```diff
@@ -137,8 +137,8 @@ struct AssessmentView: View {
                         .progressViewStyle(CircularProgressViewStyle(tint: Color.white))
                 }
             }
-            if !vm.readOnly {
-                ToolbarItem(placement: .navigationBarTrailing) {
+            ToolbarItem(placement: .navigationBarTrailing) {
+                if !vm.readOnly {
```
Comment (pal03377) [inline Themis/Views/Assessment/AssessmentView.swift]: Is there a reason for the switch here? It worked before and looked more clean imo

**Review by pal03377 — APPROVED** [review]
Apart from the small thing that @AlexExcellence mentioned, I really like this solution!

File: Themis/Views/Assessment/AssessmentView.swift (hunk 3 above)
Comment (crustypluto19) [reply Themis/Views/Assessment/AssessmentView.swift]: I grouped them before to make the animation smoother, I'll revert it back to what it was before as the grouping is no longer needed.

**Pull request merged**